src/api/main.py

```diff
@@ -299,15 +300,17 @@ def json_root():
             "/json/shots",
             "/json/cpf_summary",
             "/json/scenarios",
-            "/json/sources",  
-        ]
+            "/json/sources",
+        ],
     }
 
+
 @app.get(
     "/json/shots",
     description="Get information about experimental shots",
     response_model=CursorPage[models.ShotModel],
     response_class=CustomJSONResponse,
+    response_model_exclude_none=True,
```

Comment:
I am not sure how this fixes the issue. Doesn't this just drop items that are `None`?